tools/ValidateInputFile.py

```diff
@@ -99,14 +99,14 @@ def validate_input_file(
     )
 
     if process.returncode == 0:
+        rich.print(f"Input file '{input_file_path}' parsed successfully.")
```

Comment:
Print this at the end of the `validate_input_file_command`, not here. Never just print things in a Python functions, that's very annoying if you call the function and can't turn off the printing (we just had this in #5058, remember?). Use `logging` so at least the verbosity can be controlled from the outside. But in this case the printing is part of the CLI, not the function, so just move it to the command below.